src/components/App.tsx

```diff
@@ -5008,7 +5008,7 @@ class App extends React.Component<AppProps, AppState> {
       return;
     }
 
-    const file = event.dataTransfer?.files[0];
+    const file: File | undefined = event.dataTransfer?.files[0];
```

Comment:
```suggestion
    const file = event.dataTransfer?.files.item(0);
```

Reply:
Doing `: File | undefined` is one of the rare cases where type declaration is less typesafe than doing a type assertion `as File | undefined`, because TS will narrow down the assignment as `File` since it's inferred as such.

But, I've changed it to using `.item()` API instead which has the correct type we need.